cypress/support/step_definitions/html.js

```diff
@@ -142,8 +142,9 @@ Then('I expect {string} height is {int}', (templateSelector, height) => {
 Then('I expect {string} to be at position {int},{int}', (templateSelector, x, y) => {
```

Comment:
Rule is already use to check on screen diagrams position at diagrams page.
Prefere to have an another rules like 
`I expect component {string} to be at position {int},{int}`

Reply:
done